app/models/phase.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: phases
+#
+#  id                     :bigint           not null, primary key
+#  uuid                   :uuid             not null, default: -> { "gen_random_uuid()" }
+#  title                  :string
+#  start_date             :datetime
+#  end_date               :datetime
+#  open_to_submissions     :boolean
+#  judging_criteria        :string
+#  judging_criteria_delta  :string
+#  judging_criteria_length :integer          virtual
+#  how_to_enter            :string
+#  how_to_enter_delta      :string
+#  how_to_enter_length     :integer          virtual
+#  delete_phase            :boolean          virtual
+#  challenge_id            :bigint           not null
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+class Phase < ApplicationRecord
+  belongs_to :challenge
+  # More relations from phoenix app
+  # has_many :submissions
+  # has_one :winner, class_name: 'PhaseWinner'
+
+  # Attributes
+  attribute :uuid, :uuid
+  attribute :title, :string
+  attribute :start_date, :datetime
+  attribute :end_date, :datetime
+  attribute :open_to_submissions, :boolean
+  attribute :judging_criteria, :text
+  attribute :how_to_enter, :text
+  attribute :challenge_id, :uuid
+
+  # Virtual fields
+  attribute :judging_criteria_length, :integer, default: 0
+  attribute :how_to_enter_length, :integer, default: 0
+  attribute :delete_phase, :boolean, default: false
```

Comment:
what is this for?

Reply:
These were virtual fields from the phoenix app. Probably not needed here so I can remove them. 

`delete_phase` specifically was used on submission of that step in the challenge wizard to delete any phases with that delete checkbox checked. Probably somewhat vestigial and can might be or could be done in a better way on the phoenix app.

Reply:
oh yeah, I haven't even tried setting up multiple phases on a challenge yet 🙃